src/score_hv/harvesters/daily_bfg.py

```diff
@@ -2,32 +2,35 @@
 
 import os
 import sys
-from pathlib import Path
-from collections import namedtuple
-from dataclasses import dataclass
-from dataclasses import field
-from datetime import datetime as dt
-
+import ast
 import xarray as xr
 import numpy as np
 import cftime
+from pathlib import Path
 from netCDF4 import MFDataset
+from datetime import datetime as dt
+from collections import namedtuple
+from dataclasses import dataclass
+from dataclasses import field
+from score_hv.config_base  import ConfigInterface
+from score_hv.stats_utils  import var_stats
+from score_hv.region_utils import GeoRegions
 
-from score_hv.config_base import ConfigInterface
-from score_hv import stats_utils
 
 HARVESTER_NAME = 'daily_bfg'
 VALID_STATISTICS = ('mean', 'variance', 'minimum', 'maximum')
 
+
```

Comment:
should probably hook up support to verify that requested bound keys are valid

```suggestion
VALID_REGION_BOUND_KEYS = ('min_lat', 'max_lat', 'east_lon', 'west_lon')
```